src/widgets/hotNews/HotNews.tsx

```diff
@@ -0,0 +1,39 @@
+import Card from "../../entities/card";
+import styles from './hot-news.module.css'
+import { Spinner } from "~shared/ui/spinner";
+
+export function HotNews() {
+  const demoData = {
```

Comment:
at least move it out of component please

Reply:
not fixed

Reply:
fixed